host/libhackrf/src/hackrf.h

```diff
@@ -66,6 +66,7 @@ enum hackrf_board_id {
 	BOARD_ID_JELLYBEAN  = 0,
 	BOARD_ID_JAWBREAKER = 1,
 	BOARD_ID_HACKRF_ONE = 2,
+	BOARD_ID_RAD1O = 3,
```

Comment:
It might be that the original FW did not do that. We got our own USB PID just before camp and might have changed the board ID around that time.